### PR TITLE
12.0 website no crawler

### PR DESCRIPTION
**Pull request by smurf-U**

### CODE REVIEW

**Comment by nikul-serpentcs** [comment]
@smurf-U In Readme, use code-genreadme like this https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

**Review by tarteo — APPROVED** [review]

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM :+1:

File: website_google_tag_manager/README.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
-==================
```
Comment (simahawk) [inline website_google_tag_manager/README.rst:1]: hmmm what has to deal this w/ no crawler? :thinking:

**Review by simahawk — CHANGES_REQUESTED** [review]

File: website_google_tag_manager/README.rst (hunk 1 above)
Comment (smurf-U) [reply website_google_tag_manager/README.rst:1]: @simahawk Nothings..
But because of that Travis failed.
After change it done

Comment (simahawk) [reply website_google_tag_manager/README.rst:1]: if you need this change to fix this build, pls, split it to a specific commit like `website_google_tag_manager: fix RST`

**Review by atchuthan — APPROVED** [review]
:+1: 
It would be good to have this PR merged

**Comment by pedrobaeza** [comment]
The comment from @simahawk is not attended.

**Review by fmdl — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@simahawk can you re-review this?

**Review by simahawk — APPROVED** [review]

**Pull request merged**